lib/node_modules/@stdlib/blas/ext/base/dsapxsumpw/README.md

```diff
@@ -36,27 +36,27 @@ limitations under the License.
 var dsapxsumpw = require( '@stdlib/blas/ext/base/dsapxsumpw' );
 ```
 
-#### dsapxsumpw( N, alpha, x, stride )
+#### dsapxsumpw( N, alpha, x, strideX )
 
-Adds a constant to each single-precision floating-point strided array element and computes the sum using pairwise summation with extended accumulation and returning an extended precision result.
+Adds a constant to each single-precision floating-point strided array element, computes the sum using pairwise summation with extended accumulation and returning an extended precision result.
```

Comment:
In #3129 "Computes the sum of single-precision floating-point strided array elements, ignoring `NaN` values,(*) using ordinary recursive summation with extended accumulation and alternative indexing semantics, and returning an extended precision result."

* we removed the `and` and used `,` instead because of `ignoring NaN`?

Reply:
Yes, there, it is a different sentence construction, where `ignoring...`, `using...`, and `returning...` form a list. In the current description 

> computes the sum using pairwise summation with extended accumulation and returning an extended precision result

...is an independent sentence.

I suppose we should make this more clear with

> Adds a constant to each single-precision floating-point strided array element, and computes the sum using pairwise summation with extended accumulation and returning an extended precision result.

Namely, `, and computes`.